src/Data/Traversable.purs

```diff
@@ -60,6 +62,39 @@ sequenceDefault :: forall t a m. (Traversable t, Applicative m) =>
 sequenceDefault tma = traverse id tma
 
 
+newtype Id a = Id a
+
+runId :: forall a. Id a -> a
+runId (Id a) = a
```

Comment:
Perhaps we should use `purescript-identity` here, and flip the dependency order?